app.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+var allLocations = [ ];
+
+var hoods = [
+  'Downtown',
+  'Belltown',
+  'Pioneer Square',
+  'Capitol Hill',
+  'First Hill',
+  'SLU',
+  'U District',
+  'Uptown',
+  'Greenlake',
+  'SODO'
+];
+
+//Object constructor for locations
+var Location = function(obj) {
+  this.name = obj.name;
+  this.hood = obj.hood;
+  this.address = obj.address;
+  this.mealType = obj.meal;
+  this.daysOpen = obj.days;
+  this.openTime = obj.openTime;
+  this.closeTime = obj.closeTime;
+  this.restrictions = obj.restrictions;
+  this.reservations = obj.reservations || 0;
+  allLocations.push(this);
+};
+
+function instantiateLocations() {
+  for(var i = 0; i < breakfastLocationData.length; i++) {
+    new Location(breakfastLocationData[i]);
+    new Location(lunchLocationData[i]);
+    new Location(dinnerLocationData[i]);
+  }
+};
+
+instantiateLocations();
+
+var map;
+
+function googleMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 47.6050, lng: -122.3344},
+    zoom: 13
+  });
+};
+
+// start filtering meal type locations IF CLICKED. saving for now just in case
```

Comment:
it's still best to omit commented code from the master branch. You can always keep it on a feature branch and copy it from there to master if you want to un-comment it.